### DISCUSSION TITLE
Error "Prerequisite MinNT-20170416-85fac4faadc77203db8ddc66af280a75c1b717b0.zip not found"

### DISCUSSION THREAD

**Discussion by Solstar** [leecher1337/ntvdmx64#249] [General]
hi all just got the latest rel from 20/10/2023, i am running autoupdate all but at some point i have this error
Scanning the drive for archives:
1 file, 46885057 bytes (45 MiB)

Extracting archive: C:\dwn\\NTOSBE-master.zip
--
Path = C:\dwn\\NTOSBE-master.zip
Type = zip
Physical Size = 46885057
Comment = 8a3baea4de52c5c9f3d97d58857e35018edc0a64

Everything is Ok

Folders: 345
Files: 2968
Size:       139245880
Compressed: 46885057
        1 file spostato/i.
Prerequisite MinNT-20170416-85fac4faadc77203db8ddc66af280a75c1b717b0.zip not found in C:\dwn\, FAILED!
Premere un tasto per continuare . . .

**Comment by leecher1337** [comment]
So are you sure you put this file into your build directory?

**Comment by Solstar** [comment]
Everything was there as required yet it didn't work with the error yo usaw

**Comment by leecher1337** [comment]
I doubt it, the error message is issued when 
`call :unpack MinNT-20170416-85fac4faadc77203db8ddc66af280a75c1b717b0.zip`
failed und this just checks:
```
:unpack
if not exist %PREREQ%\%1 (
  echo Prerequisite %1 not found in %PREREQ%, FAILED!
  pause
  exit
)
```

Maybe the minnt zip file you put there is named slightly differently? 
You can check with

`dir c:\dwn\MinNT-20170416-85fac4faadc77203db8ddc66af280a75c1b717b0.zip`

to check for its presence.

**Comment by Solstar** [comment]
I have this in the folder see screenshot attached

**Comment by leecher1337** [comment]
I don't see a screenshot, there is nothing attached to your post

**Comment by Solstar** [comment]
try now

**Comment by Solstar** [comment]
Have you seen the screenshot now? if not here's a link to imgur https://imgur.com/a/VujROnf

**Comment by leecher1337** [comment]
And where is the `MinNT-20170416-85fac4faadc77203db8ddc66af280a75c1b717b0.zip` in your screenshot? 
Because I cannot see it there?
Do you want to troll me or what?

**Comment by Solstar** [comment]
That was another screenshot , why don't you calm down rather than assuming i am trolling you, you know what nevermind i don't care about this thing anymore at this point 
--
Inviato da Libero Mail

**Comment by leecher1337** [comment]
Which screenshot? The https://imgur.com/a/VujROnf doesn't show an image with the required file, there are no other images attached to this post, as you can  clearly see yourself on https://github.com/leecher1337/ntvdmx64/discussions/249 
so what do you expect me to assume?

**Comment by Solstar** [comment]
Whatever Sorry for bothering you, you clearly woke up from the wrong side of the bed. Can't understand why must be so difficult to do It now when a couple of years ago was done in two steps.

Again Sorry for ruining your precious day
--
Inviato da Libero Mail
